<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_frame</class>
 <widget class="QFrame" name="home_frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QFrame" name="side_frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border: 0px;
	border-radius: 5px;
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="homePage_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#homePage_label{
	padding-left: 5px;
	color: rgb(26, 194, 161);
}</string>
        </property>
        <property name="text">
         <string>Browse Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="pageBtn_frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>220</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#pageBtn_frame{
	border-top: 1px solid rgb(170, 170, 170);
	border-radius: 0px;
}
QPushButton{
	border: 1px solid rgb(170, 170, 170);
	border-radius: 5px;
	font: 600 10pt &quot;Segoe UI Semibold&quot;;
}
QPushButton:hover{
	border: 1px solid rgb(26, 194, 161);
	background-color: rgba(26, 194, 161, 30);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="homeProfileBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: rgb(100, 100, 100);
}</string>
           </property>
           <property name="text">
            <string>Student's Profile</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../.designer/assets/user.png</normaloff>../../../../../../.designer/assets/user.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="homeGradesBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: rgb(100, 100, 100);
}</string>
           </property>
           <property name="text">
            <string>Grades</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../.designer/assets/sheet.png</normaloff>../../../../../../.designer/assets/sheet.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="homeAttendanceBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: rgb(100, 100, 100);
}</string>
           </property>
           <property name="text">
            <string>Attendance</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../.designer/assets/attendance.png</normaloff>../../../../../../.designer/assets/attendance.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="main_home_frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="navBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="school_name">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(100, 100, 100);
}</string>
           </property>
           <property name="text">
            <string>INTERNATIONAL HIGH SCHOOL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	padding-left: 15px;
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>Dashboard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QFrame" name="frame_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
	border: 0px;
	border-radius: 10px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>10</number>
               </property>
               <item row="1" column="0" colspan="2">
                <widget class="QFrame" name="classGenderStat_3">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	border: 0px;
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QFrame" name="classGenderStat">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>230</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	border: 0px;
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QFrame" name="classGenderStat_4">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>230</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	border: 0px;
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="subjectStat">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>230</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
	border: 0px;
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QPushButton" name="btnCreateNewClass">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>310</y>
                    <width>131</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:focus{
color: rgb(0, 0, 0);
	border-radius: 5px;
	border: 2px solid rgb(26, 194, 161);
	background-color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(19, 140, 116);
}
QPushButton{
background-color: rgb(26, 194, 161);
	color: rgb(255, 255, 255);
 	border-radius: 5px;
}</string>
                  </property>
                  <property name="text">
                   <string>Create new class</string>
                  </property>
                 </widget>
                 <widget class="QComboBox" name="comboBox">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>180</y>
                    <width>121</width>
                    <height>31</height>
                   </rect>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_2">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>20</y>
                    <width>121</width>
                    <height>91</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
	color: rgb(100, 100, 100);
}</string>
                  </property>
                  <property name="text">
                   <string>Grade 8 - Amethyst 2022 -2023</string>
                  </property>
                  <property name="scaledContents">
                   <bool>false</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>frame_2</zorder>
     <zorder>navBar</zorder>
    </widget>
   </item>
  </layout>
  <zorder>main_home_frame</zorder>
  <zorder>side_frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
